luthier/config/src/utils/tableUtils.lua

```diff
@@ -12,33 +12,72 @@ local misc = require 'utils./misc'
 -- Build Config
 -------------------------------------------------------------------------------
 
+--- Build a specific physical table into a json object.
+--
+-- for each of the pair in logicalToPhysicalColumnNames, e.g.
```

Comment:
This is going to need some sort of context and/or transition. The
logicalToPhysicalColumnNames is only one part of the transformation, but
the documentation reads like this is the only transformation. To people
who aren't familiar with the codebase, this could be misleading. To those
who are, this reads like a non-sequitor.

You should start off with something like `Most of the transformations
are straightforward (i.e. initializing defaults), however
`logicalToPhysicalColumnNames` is converted into a rather weird
format.`

Reply:
Michael and I had the concept earlier to use a separate lua file to supply all defaulting parameter for the user to see. Due to that fact I will leave the default live in the table.lua and not mention them in tableUtils.lua (and all corresponding ones), for now. 

Also addressed your comments

Reply:
Consolidating all the defaults in a single file is a very good idea.